server/utils/users/user.util.js

```diff
@@ -64,36 +49,47 @@ class userValidation {
   }
 
   /**
-   * User object empty check
+   * User sign in object empty check
    * @param {Object} req
    * @param {Object} res
    * @param {Object} next
    * @returns {Object} response
    */
-  static userEmpty(req, res, next) {
-    const {
-      email,
-      password,
-      firstname,
-      lastname
-    } = req.body;
-    return isEmpty({
-      email,
-      password,
-      firstname,
-      lastname
-    }, res, next);
+  static async isSigninFieldEmpty(req, res, next) {
+    // const { email, password } = req.body;
```

Comment:
kindly remove the comment on this line @davidokonji

Reply:
Thank you for the observation, would fixed that now